panels/history/ha-panel-history.html

```diff
@@ -41,13 +48,22 @@
       </app-header>
 
       <div class="flex content">
-        <paper-input
-          label='Showing entries for'
-          id='datePicker'
-          value='[[_computeDateDisplay(_selectedDate)]]'
-          on-focus='datepickerFocus'
-        ></paper-input>
-
+        <div class="flex layout horizontal wrap">
+          <paper-input
+            label='Start date'
+            id='datePicker'
+            value='[[_computeDateDisplay(_selectedDate)]]'
+            on-focus='datepickerFocus'
+          ></paper-input>
+          <paper-dropdown-menu label-float label='Period'>
+            <paper-menu class="dropdown-content" selected="{{_periodIndex}}">
+              <paper-item>1 day</paper-item>
+              <paper-item>1 week</paper-item>
+              <paper-item>30 days</paper-item>
```

Comment:
Since we do not do any filtering or consolidation of events in the backend, this will break pretty much every machine out there.

Imagine having just 2 Z-Wave sensors that soon both will have force_update and report every 5 minutes. That's 24 reports an hour, 576 per day. So 30 days will bring result in a total of 12,280 events and will totally kill the browser. And that's with just 2 sensors!

May I suggest we go for 1 day, 3 days and 1 week initially.

In the future we can consider expanding this once we do event consolidation on the backend (either on the fly or we maintain a "simplified" event stream in another table that filters out glitches and reduces reports to something like once an hour)